jupyter.opam

```diff
@@ -17,7 +17,7 @@ build: [
   [ "dune" "build" "-p" name "-j" jobs ]
 ]
 depends: [
-  "ocaml" {>= "4.10.0" & < "4.15"}
+  "ocaml" {(>= "4.10.0" & < "4.15") | (>= "5.0.0")}
```

Comment:
I forget updating the restriction of ocaml versions. I'd like to suggest

```
"ocaml" {>= "4.10.0"}
```

because ocaml-jupyter kernel now supports 4.15.
cf. [jupyter.2.8.2/opam](https://github.com/ocaml/opam-repository/blob/master/packages/jupyter/jupyter.2.8.2/opam)